head.html

```diff
@@ -134,6 +134,36 @@
     }
   }
 
+  function fetchProducts() {
+    const div = getSection();
+    const insertAfter = getSection(2);
+    if (insertAfter) {
+      const r = /^Products\: (.*)$/gm.exec(div.innerText);
+      const products = r.length > 0 ? r[1].split(', ') : null;
+      if (products) {
+        div.innerText = div.innerText.replace(r[0], '');
```

Comment:
```suggestion
        div.innerHTML = div.innerHTML.replace(`<p>${r[0]}</p>`, '');
```
Make sure the DOM stays intact and just the right `<p>` tag gets stripped. Otherwise, all `<p>`s  will be replaced with plain text and `<br>`s. See also `fetchTopics()` in master.

Reply:
Interestingly I did not have your described behavior before your change but now I have it...

Reply:
`fetchTopics` is the one introducing now the `<br>`s, I assume you did not commit / merge what you are talking about...

Reply:
Ok, I fixed this in a nicer way (iterating through children nodes)

Reply:
You're right, sorry, it's still in my `latest-posts` branch 😬